validator/client/service.go

```diff
@@ -188,10 +188,21 @@ func (v *ValidatorService) Start() {
 		return
 	}
 
+	// checks db if proposer settings exist if none is provided.
+	if v.proposerSettings == nil {
+		settings, err := v.db.ProposerSettings(v.ctx)
```

Comment:
what happens if err is not nil?

Reply:
it's going to start without db settings, will start with the flag settings, and if those are not provided it will fall back to the beacon node's settings